statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -308,6 +387,125 @@ def _reordered(self, order):
                       lag_order=self.k_ar, model=self.model,
                       trend='c', names=names_new, dates=self.dates)
 
+def orth_ma_rep(results, maxn=10, P=None):
+    r"""Compute Orthogonalized MA coefficient matrices using P matrix such
+    that :math:`\Sigma_u = PP^\prime`. P defaults to the Cholesky
+    decomposition of :math:`\Sigma_u`
+
+    Parameters
+    ----------
+    results : VARResults or VECMResults
+    maxn : int
+        Number of coefficient matrices to compute
+    P : ndarray (neqs x neqs), optional
+        Matrix such that Sigma_u = PP', defaults to the Cholesky decomposition.
+
+    Returns
+    -------
+    coefs : ndarray (maxn x neqs x neqs)
+    """
+    if P is None:
+        P = results._chol_sigma_u
+
+    ma_mats = results.ma_rep(maxn=maxn)
+    return mat([np.dot(coefs, P) for coefs in ma_mats])
+
+
+def test_normality(results, signif=0.05):
+    """
+    Test assumption of normal-distributed errors using Jarque-Bera-style
+    omnibus Chi^2 test
+
+    Parameters
+    ----------
+    results : VARResults or statsmodels.tsa.vecm.vecm.VECMResults
+    signif : float
+        The test's significance level.
+
+    Notes
+    -----
+    H0 (null) : data are generated by a Gaussian-distributed process
+
+    Returns
+    -------
+    result : NormalityTestResults
+        
+    References
+    ----------
+    .. [1] Lutkepohl, H. 2005. *New Introduction to Multiple Time Series Analysis*. Springer.
+    .. [2] Kilian, L. & Demiroglu, U. (2000). *Residual-Based Tests for Normality in Autoregressions: Asymptotic Theory and Simulation Evidence*. Journal of Business & Economic Statistics
+    """
+    resid_c = results.resid - results.resid.mean(0)
+    sig = np.dot(resid_c.T, resid_c) / results.nobs
+    Pinv = npl.inv(chol(sig))
+
+    w = np.dot(Pinv, resid_c.T)
+    b1 = (w**3).sum(1)[:, None] / results.nobs
+    b2 = (w**4).sum(1)[:, None] / results.nobs - 3
+
+    lam_skew = results.nobs * np.dot(b1.T, b1) / 6
+    lam_kurt = results.nobs * np.dot(b2.T, b2) / 24
+
+    lam_omni = float(lam_skew + lam_kurt)
+    omni_dist = stats.chi2(results.neqs * 2)
+    omni_pvalue = float(omni_dist.sf(lam_omni))
+    crit_omni = float(omni_dist.ppf(1 - signif))
+
```

Comment:
I have checked the implementation in `sm.stats.stattools`. It relies on `scipy` and `scipy`'s code differs substantially from the test proposed in Lütkepohl (2005). To me it looks as if there exist several proposals of such a test in literature which are all based on the same asymptotics but differ nonetheless. I would suggest not to use `sm.stats.stattools` in this case because the reference software _JMulTi_ seems to rely on Lütkepohl's formulas. Changing to `sm.stats.stattools` would break the corresponding test cases (I tried it out).